xarray/core/formatting_html.py

```diff
@@ -382,16 +382,40 @@ def summarize_datatree_children(children: Mapping[str, DataTree]) -> str:
     expand_option_name="display_expand_groups",
 )
 
+inherited_coord_section = partial(
+    _mapping_section,
+    name="Inherited coordinates",
+    details_func=summarize_coords,
+    max_items_collapse=25,
+    expand_option_name="display_expand_coords",
+)
+
+
+def datatree_node_repr(group_title: str, node: DataTree, show_inherited=False) -> str:
```

Comment:
I assume this logic now basically matches that in `datatree_repr`?

Reply:
That's the idea yes